{{cookiecutter.project_slug}}/clients/mobile/react-native/package.json

```diff
@@ -5,93 +5,92 @@
+    "run:android:release": "expo run:android --variant release",
+    "run:ios": "expo run:ios",
+    "run:ios:release": "expo run:ios --configuration Release",
     "pub:web": "npx expo export:web && cd web-build && vercel --prod && cd ..",
     "format": "prettier --check ./src",
     "format:write": "prettier --write ./src"
   },
   "dependencies": {
-    "@react-native-async-storage/async-storage": "1.17.11",
+    "@react-native-async-storage/async-storage": "1.18.2",
     "@react-native-community/masked-view": "^0.1.11",
-    "@react-native-picker/picker": "2.4.8",
+    "@react-native-picker/picker": "2.4.10",
     "@react-navigation/bottom-tabs": "^6.4.0",
     "@react-navigation/drawer": "^6.5.8",
     "@react-navigation/material-top-tabs": "^6.5.3",
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.0",
     "@react-navigation/stack": "^6.3.0",
-    "@sentry/react-native": "4.13.0",
-    "@shopify/flash-list": "1.4.0",
+    "@react-spring/native": "^9.7.3",
+    "@sentry/react-native": "5.5.0",
+    "@shopify/flash-list": "1.4.3",
     "@tanstack/react-query": "^4.24.4",
     "@thinknimble/tn-forms": "^3.1.3",
     "@thinknimble/tn-forms-react": "^1.0.3",
     "@thinknimble/tn-models-fp": "2.5.3",
     "axios": "^1.3.2",
     "date-fns": "^2.29.3",
-    "expo": "^48.0.9",
-    "expo-application": "~5.1.1",
-    "expo-build-properties": "~0.5.1",
-    "expo-dev-client": "~2.1.6",
-    "expo-device": "~5.2.1",
-    "expo-font": "~11.1.1",
-    "expo-linear-gradient": "~12.1.2",
-    "expo-localization": "~14.1.1",
-    "expo-notifications": "~0.18.1",
-    "expo-splash-screen": "~0.18.1",
-    "expo-status-bar": "~1.4.4",
-    "expo-updates": "~0.16.3",
-    "expo-web-browser": "~12.1.1",
-    "i18n-js": "^4.1.1",
+    "expo": "^49.0.0",
+    "expo-application": "~5.3.0",
+    "expo-build-properties": "~0.8.3",
+    "expo-dev-client": "~2.4.8",
+    "expo-device": "~5.4.0",
+    "expo-font": "~11.4.0",
+    "expo-linear-gradient": "~12.3.0",
+    "expo-localization": "~14.3.0",
+    "expo-notifications": "~0.20.1",
+    "expo-splash-screen": "~0.20.5",
+    "expo-status-bar": "~1.6.0",
+    "expo-updates": "~0.18.12",
+    "expo-web-browser": "~12.3.2",
+    "i18n-js": "4.2.0",
     "nativewind": "^2.0.11",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-native": "0.71.4",
+    "react-native": "0.72.4",
     "react-native-dotenv": "^3.4.8",
-    "react-native-gesture-handler": "~2.9.0",
-    "react-native-pager-view": "6.1.2",
-    "react-native-reanimated": "~2.14.4",
-    "react-native-safe-area-context": "4.5.0",
-    "react-native-screens": "~3.20.0",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-pager-view": "6.2.0",
+    "react-native-reanimated": "~3.3.0",
+    "react-native-safe-area-context": "4.6.3",
+    "react-native-screens": "~3.22.0",
     "react-native-tab-view": "^3.3.4",
-    "react-native-web": "~0.18.11",
-    "rn-bounceable": "^1.2.0",
```

Comment:
Seems like this package was deleted as a mistake, after installing it everything works fine 👍🏻